.gitignore

```diff
@@ -46,3 +46,4 @@ dmmf.json
 graph.dot
 
 prisma-schema-wasm/nodejs
+*.sh
```

Comment:
Should some specific generated file be here instead of the wildcard? There's a bunch of `.sh` files in the repository right now and it will make it not possible to commit changes to them if added here.

Reply:
Definitely, this was a left over.